samcli/cli/cli_config_file.py

```diff
@@ -29,44 +28,41 @@ class TomlProvider:
     def __init__(self, section=None):
         self.section = section
 
-    def __call__(self, file_path, config_env, cmd_name):
+    def __call__(self, config_dir, config_env, cmd_names):
         """
         Get resolved config based on the `file_path` for the configuration file,
         `config_env` targeted inside the config file and corresponding `cmd_name`
         as denoted by `click`.
 
-        :param file_path: The path to the configuration file
         :param config_env: The name of the sectional config_env within configuration file.
-        :param cmd_name: sam command name as defined by click
+        :param cmd_names list(str): sam command name as defined by click
         :returns dictionary containing the configuration parameters under specified config_env
         """
+
         resolved_config = {}
-        try:
-            config = toml.load(file_path)
-        except Exception as ex:
-            LOG.error("Error reading configuration file :%s %s", file_path, str(ex))
+
+        samconfig = SamConfig(config_dir)
```

Comment:
Is there any ability for users to pass a custom file name? As far as I can tell, we don't have a code path for that here.

Reply:
In the future, not now.